docs/data/material/getting-started/templates/templates.md

```diff
@@ -40,9 +44,13 @@ You can find complete templates and themes like those shown below in the <a href
 
 ## Toolpad Core (beta)
 
-[Toolpad Core](https://mui.com/toolpad/core/introduction/) is a set of components designed for assembling dashboards and internal tools more efficiently.
-It builds on top of Material UI to provide sophisticated UI components with full-stack functionality, such as the [Dashboard Layout](https://mui.com/toolpad/core/react-dashboard-layout/) shown below.
-
-This is a great option if you don't want to start building from scratch but don't need a prebuilt template.
+If you're looking to build internal tools and dashboards fast, but don't want to start building from scratch, you can find examples of functional dashboards – with authentication, routing, and theming already integrated – in the [featured examples](http://mui.com/toolpad/core/introduction/examples/#featured-examples) section of the Toolpad Core docs.
 
-{{"demo": "../../components/app-bar/DashboardLayoutBasic.js", "height": 400, "iframe": true, "defaultExpanded": false, "bg": "inline"}}
+<a href="https://mui.com/toolpad/core/introduction/examples/#featured-examples">
+<span class="only-light-mode">
+<img src="/static/images/toolpad-core-template-light.png" alt="The Toolpad Core template comes with authentication, navigation and theming integrated" width="2280" height="1200" style="display: block;" />
```

Comment:
The width and height are wrong. It says ` width="2280" height="1200"` but the actual image is 2400x1362. This is not the same aspect ratio. This leads to layout shifts.